src/api/spec/features/webui/packages_spec.rb

```diff
@@ -96,4 +97,39 @@
     expect(page).to have_text("In state review")
     expect(page).to have_text("Set the devel project to package #{third_project.name} / develpackage for package #{user.home_project} / develpackage")
   end
+
+  context "triggering package rebuild" do
+    let(:repository) { create(:repository, architectures: ["x86_64"]) }
+    let(:rebuild_url) {
```

Comment:
VCR would only mock the request, but require it to happen. The only other option I see here would be to set up an expect on the package object.